src/config.js

```diff
@@ -1,7 +1,16 @@
 const config = {
 	wikiWhoUrl: 'https://www.wikiwho.net/',
 	// Set the output type between the extension and the gadget
-	outputEnvironment: ''
+	outputEnvironment: '',
+	// Elements that we know for sure are never tokenized and should be grayed out.
+	untokenizedElements: [
+		'.citation',
+		'.IPA',
+		'.mw-cite-backlink',
+		'.mw-editsection',
+		'.reference',
+		'table'
```

Comment:
I'm not really sure why, but on Firefox, a table caption (like 'Club statistics' on [this page](https://en.wikipedia.org/wiki/Kevin_Ram%C3%ADrez_(footballer,_born_1993))) is not greyed out. It works correctly when I add 'caption' to this list, but the opacity goes weird (because it's compounding, I think).

Reply:
I'm not sure what to do here. I tried `caption, tr` and `table > *` etc., but you either end up with the compounding opacity or the `:hover` acts on just the `tr`s / `caption`s, when (I think) we'd rather the hover work on the whole table as a single entity, if that makes sense.

I found this bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1286803 There are arguments on whether `caption` should get the same opacity as the parent, but the bug is still open after 3 years so I take it that it is in fact a bug. Maybe we could live with it? There are surely other elements that don't get grayed out either (in any browser), though I guess captions are fairly common.

In working on this I did find situations where there was compounded opacity (e.g. table inside of a table), and I think I have that fixed.